scripts/pdf/README.md

```diff
@@ -17,13 +17,13 @@ This directory contains the script and related resources to generate a PDF copy
 
 ## Requirements
 
-The PDF is generated by first converting the markdown files to HTML, and then rendering those HTML files as PDF. It depends on the following libraries-
+The PDF is generated by first converting the markdown files to HTML, and then rendering those HTML files as PDF. It depends on the following libraries:
 
-#### Python-Markdown
+**Python-Markdown**
```

Comment:
Let us revert these. `####` is a correct markdown syntax and is consistent with `##` above.

Reply:
Alright sure, I have allowed edits to be done from maintainers.
Will you edit the PR or should I do it?